paasta_tools/deployd/workers.py

```diff
@@ -102,7 +105,7 @@ def run(self) -> None:
                         base=2,
                         max_time=6000,
                     )
-                if bounce_again_in_seconds:
+                if bounce_again_in_seconds and failures < self.max_failures:
```

Comment:
I think this would be more helpful like:
```
if max_fail:
  _log("Failed N times in a row trying to deploy this service. Refusing to continue.")
elfi max_attempts:
  _log("Tried to deploy this service N times. It hasn't failed but isn't making any progress. Backing off and enquing an hour later)
elsif bounce_again:
...
```

Reply:
Yeah I can log more